handler/src/main/java/io/netty/handler/ssl/OpenSsl.java

```diff
@@ -381,7 +393,27 @@
             TLSV13_SUPPORTED = false;
             IS_BORINGSSL = false;
             EXTRA_SUPPORTED_TLS_1_3_CIPHERS = EmptyArrays.EMPTY_STRINGS;
+            EXTRA_SUPPORTED_TLS_1_3_CIPHERS_STRING = StringUtil.EMPTY_STRING;
+        }
+    }
+
+    static String checkTls13Ciphers(InternalLogger logger, String ciphers) {
+        if (IS_BORINGSSL && !ciphers.isEmpty()) {
+            assert EXTRA_SUPPORTED_TLS_1_3_CIPHERS.length > 0;
+            Set<String> boringsslTlsv13Ciphers = new HashSet<String>();
+            Collections.addAll(boringsslTlsv13Ciphers, EXTRA_SUPPORTED_TLS_1_3_CIPHERS);
+            for (String cipher: ciphers.split(",")) {
+                boringsslTlsv13Ciphers.remove(cipher);
+            }
+            if (!boringsslTlsv13Ciphers.isEmpty()) {
```

Comment:
Current logic prints a message only if `EXTRA_SUPPORTED_TLS_1_3_CIPHERS` contains some ciphers that were not present in `ciphers`, but it won't print anything if `ciphers` contains everything from `EXTRA_SUPPORTED_TLS_1_3_CIPHERS` and more.

Reply:
@idelpivnitskiy I think we should not print if we exactly specify what `EXTRA_SUPPORTED_TLS_1_3_CIPHERS` contains. But yeah we should print if there is more.